src/UrlSearchParams.ts

```diff
@@ -115,19 +118,20 @@ export default class UrlSearchParams {
 	 * @param key The key to return the first value for
 	 * @return The first string value for the key
 	 */
-	get(key: string): string {
+	get(key: string): string | null | undefined {
```

Comment:
Again, the change I don't argue with, but `null` seems inconsistent for `get()`...  I think people would expect `undefined`.